docs/releases/202309.mdx

```diff
@@ -4,6 +4,12 @@ title: September 2023
 displayed_sidebar: releaseSideBar
 ---
 
+### September 21 2023
+**Improvements**
+- Confirmation is now present when updating the base version of an Agent.
+- Editor notebook cells that contain user-defined code are now colored light blue.
```

Comment:
not sure if this is confusing - it's the saved code since I guess all code in the notebook is user-defined code

Reply:
Sounds good, I updated to say user-defined function code